tools/publisher/src/package.rs

```diff
@@ -60,13 +60,16 @@ pub struct Package {
     pub manifest_path: PathBuf,
     /// Dependencies used by this package
     pub local_dependencies: BTreeSet<PackageHandle>,
+    /// Whether or not the package should be published
+    pub publish_enabled: bool,
```

Comment:
I might consider an enum for this, `publish_enabled` seems _pretty_ clear, but I wouldn't be surprised if someone gets it backwards somewhere